js-extensions/src/ExtensionRenderer.jsx

```diff
@@ -1,7 +1,7 @@
 var React = require('react');
 var ReactDOM = require('react-dom');
-var ExtensionStore = require('./ExtensionStore.js').instance;
-var ResourceLoadTracker = require('./ResourceLoadTracker').instance;
+var importedExtensionStore = require('./ExtensionStore.js').instance;
+var importedResourceLoadTracker = require('./ResourceLoadTracker').instance;
```

Comment:
Nothing to do with this PR, but I vomit into my mouth every time I see that `.instance` thing on those `js-extensions` types. Just say'n.